src/views/InteractiveMap/Card.vue

```diff
@@ -181,6 +181,20 @@
       <v-container fluid class="pa-0">
         <v-layout wrap>
           <v-flex>Growth rate:</v-flex>
+          <v-flex>
+            <div v-if="card.growthRate !== null">
+              <v-tooltip v-if="card.growthRate <= 0.00001" bottom>
```

Comment:
The warning should trigger whenever there are proteomics data + no growth rate provided, regardless of the growth rate value. That way we don't have to decide on a threshold. Sorry if this was not clear.

Reply:
It should now only trigger whenever no growth rate is provided, and only for data-driven cards with proteomics data provided. Also I moved the icon to the right.